docs/portal.properties-Reference.md

```diff
@@ -131,3 +131,25 @@ To change the gene sets used for gene querying, create a JSON file and add gene
 ```
 querypage.setsofgenes.location=file:/<path>
 ```
+# Reference Genome information for validating the genomic data
+The reference genome information will be used by the importer validation script to ensure the version of Genome Reference Consortium Build is the same version that was used to analyse the genomic data. 
+Currently, the portal only supports multiple reference genome builds for single species.
+
+1. Use **species** to indicate the organism of the reference genome. For example, **human** is for Genome Reference Consortium human build 37.
+2. Use **genome.name** to indicate the name of reference genome as used by the UCSC browser. For example, **hg19** is the name for Genome Reference Consortium human build 37. 
+3. Use **genome.build** to specify the version of Genome Reference Consortium Build as published by the NCBI. For example, **GRCh37** is Genome Reference Consortium Human Build 37.
+4. Use a comma to separate multiple genome names and builds. 
+5. Each genome name should have a matched genome build.
+
+**species, genome.name, genome.build** properties need to be added or updated in each of the following cases:
+1. After reference genomes are imported to the database (details on how to import reference genomes can be found in [this document](import-reference-genome.md))
+2. After existing database is migrated to a new schema version using migration.sql (details on how to update your existing cBioportal can be found in [this document](Updating-your-cBioPortal-installation.md)) 
+3. After new database is created from a seed database that contains these reference genomes (details on how to load seed databases can be found in [this document](Import-the-Seed-Database.md)), 
+
+Below is the sample reference genome information for a human
+```
+# species and genomic information
+species=human
+genome.name=hg19,hg19,hg38
+genome.build=37,GRCh37,GRCh38
+```
```

Comment:
On second thought, @pieterlukasse and I think it's better to remove the `genome.name` and `genome.build` property. The original purpose of `genome.build` was to validate that the value in `NCBI_Build` in the MAF file corresponds with the **species** of the cBioPortal instance: Human or Mouse. It was also used to **block** `GRCh38` mutation events from being loaded into the database.

With the database solution you implemented, the valid reference genomes are now in the database. We realize that creating a web-service to provide the reference genome to the validator is out of the scope of this PR, but we think that removing this property is important. This will greatly simplify the configuration.

For now, we propose the following solution:
1. Keep the species property. This is important to keep the cBioPortal instance specific for one species at a time.
2. Remove `genome.name` and `genome.build`. 
3. Hardcode in the validator that `GRCh37` and `GRCh38` are valid `NCBI_Build` values when `species` property is `human` and `GRCm38` is valid for `mouse`. Raise an error if the value is anything else.
4. Hardcode in the validator to retrieve the correct USCS names: hg19 for GRCh37, hg38 for GRCh38, mm10 for GRCm38.

In the next pull request, the `BUILD_NAME` value from the database should be retrieved by an API call. That way,  the validator can check if the value for `NCBI_Build` in MAF corresponds to a value in the `BUILD_NAME` column in the database. Also the UCSC name could possibly be retrieved from the `NAME` column.

Reply:
I think the structure of his markdown can still be improved. I've included a sample text in [this](https://gist.github.com/sandertan/c54ca55995602c9327486ee9313cfe41) gist, assuming `genome.name` and `genome.build` are removed. This also saves a lot of explaining what and why these properties exist :) 

Additionally, could you update the table of contents at the start of this document?